### ISSUE TITLE
core-dialog: Server rendered core dialog includes stringified onDialogToggle prop

### ISSUE THREAD

**Issue by esseb** [nrkno/core-components#261]
Example server rendered markup:
```js
<core-dialog-react class="modal-dialog__dialog" onDialogToggle="function handleDialogClose() {
  // Some logic I've removed here
  props.onClose();
}" aria-label="React dialog">
 ...
</core-dialog-react>
```

This causes a React hydration warning because of a difference in my `handleDialogClose` function between server and client due to code minification etc.

Fails on master. Regressed between [feb16c6](https://github.com/nrkno/core-components/commit/feb16c6cc177ae463badeeb0f0302230777dbadb) and [0f41aab](https://github.com/nrkno/core-components/commit/0f41aab61aadaef06795dd2c1fd45a7d6a1ed2c9).

**Comment by htor** [comment]
thanks for this! looking into it..

**Comment by htor** [comment]
closed by #265
